client/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "client",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

Comment:
Thinking it would be nice to refactor this in the future so that we can run client and server concurrently